### DISCUSSION TITLE
How to change "Dear" in `coverletter.typ`?

### DISCUSSION THREAD

**Discussion by recursiveAdaption** [DeveloperPaul123/modern-cv#34] [Q&A] [answered]
how would I modify the starting word "Dear" in `coverletter.typ` or `lib.typ`?

```typst
#linguify("dear", from: lang_data) #addressee,
```
modifying line 657 in `lib.typ` unfortunately didn't change anything,
- and searching through the entire file for "dear" didn't return any results so I'm stuck at this point

**Comment by recursiveAdaption** [comment]
not sure why but it seems that initializing a new project using the command from Typst Universe `typst init @preview/modern-cv:0.3.1` causes issues where some files aren't pulled from the repository

I was informed by jimpjorps on discord that `lang.toml` would need to be edited however that file doesn't exist in the newly initialized directory

Hope this helps for anyone looking to edit a thing within this project but unable to find a reference/file

**Comment by ptsouchlos** [comment, marked as answer]
Overriding `dear` in the template has now been added and will be included in the `0.4.0` release.
